templates/ohdsi-webapi/main.bicep

```diff
@@ -0,0 +1,78 @@
+targetScope = 'resourceGroup'
+
+param location string = resourceGroup().location
+param tenantId string = subscription().tenantId
+param utc string = utcNow()
+param odhsiWebApiName string = 'ohdsi-webapi'
+param suffix string = uniqueString(utc)
+@secure()
+param postgresAdminPassword string
+@secure()
+param postgresWebapiAdminPassword string
+@secure()
+param postgresWebapiAppPassword string
+
+@description('Creates the database server, users and groups required for ohdsi webapi')
+module atlasDatabase 'atlas_database.bicep' = {
+  name: 'atlasDatabase'
+  params: {
+    location: location
+    suffix: suffix
+    odhsiWebApiName: odhsiWebApiName
+    postgresAdminPassword: postgresAdminPassword
+    postgresWebapiAdminPassword: postgresWebapiAdminPassword
+    postgresWebapiAppPassword: postgresWebapiAppPassword
+  }
+}
+
+@description('Creates the app service plan')
+module appServicePlan 'appplan.bicep' = {
```

Comment:
Shouldn't this also rename to `app_plan.bicep`?

Reply:
Done